utils/utils.ts

```diff
@@ -38,13 +38,13 @@ export function getEnvironmentRequiredVars() {
   const testUserName = process.env.TEST_USER_NAME
     ? process.env.TEST_USER_NAME
     : "//Alice";
-  if (
-    (palletAddress.length === 0 && xykPalletAddress.length === 0) ||
-    sudoUserName.length === 0 ||
-    treasuryPalletAddress.length === 0
-  ) {
-    throw new Error("PALLET ADDRESS OR SUDO USERNAME NOT FOUND AS GLOBAL ENV");
-  }
+  // if (
+  //   (palletAddress.length === 0 && xykPalletAddress.length === 0) ||
+  //   sudoUserName.length === 0 ||
+  //   treasuryPalletAddress.length === 0
+  // ) {
+  //   throw new Error("PALLET ADDRESS OR SUDO USERNAME NOT FOUND AS GLOBAL ENV");
+  // }
```

Comment:
I think we need to find a way to ensure that at least some env. variables are there. but agree that this is not the right way :)